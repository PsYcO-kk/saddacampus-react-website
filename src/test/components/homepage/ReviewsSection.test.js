import {shallow} from 'enzyme';
import React from 'react';
import ReviewSection from './../../../components/homepage/ReviewsSection.js';

test("testing for Reviews Section page component" , ()=>{
    const wrapper = shallow(<ReviewSection />)
    expect(wrapper).toMatchSnapshot();
})