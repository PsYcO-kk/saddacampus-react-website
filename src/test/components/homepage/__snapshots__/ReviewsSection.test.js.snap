// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing for Reviews Section page component 1`] = `
<section
  className="reviews-area section-gap"
>
  <div
    className="container"
  >
    <div
      className="row section-title"
    >
      <h1>
        User Reviews
      </h1>
      <p>
        Who are extremely in love with our platform.
      </p>
    </div>
    <div
      className="row"
    >
      <UserReview
        username="Shubham"
      />
      <UserReview
        username="Saurabh Kabra"
      />
      <UserReview
        username="Sparsh Choudhary"
      />
      <UserReview
        username="Jon Banks"
      />
      <UserReview
        username="Landon Houston"
      />
      <UserReview
        username="Nelle Wade"
      />
    </div>
  </div>
</section>
`;
